packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -20,9 +23,11 @@ interface AtomicArguments {
 }
 
 export default class Atomic extends Command {
+  public static cliPackage = '@coveo/create-atomic';
+  public static requiredNodeVersion = '>=8.9.4';
```

Comment:
Plop requires this, does `@coveo/search-token-server` require a specific version?

Reply:
Good question.
I was able to make it work with Node v7. However, the search token server uses dotenv v10 which, based on their documentation, requires node v10.